### PR TITLE
feat: add new language (lt-LT)

### PR DESCRIPTION
**Pull request by AndreyNegrov**
add new language (lt-LT)

### CODE REVIEW

File: locales/lt-LT/messages.po [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+"sourceLanguage: en-US\n"
+"targetLanguage: lt-LT\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Poedit-Basepath: ../../\n"
+"X-Poedit-KeywordsList: __\n"
+"X-Poedit-SearchPath-0: .\n"
+
+#, tao-public
+msgid " "
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Are we translating empty fields?

Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: i agree it's weird
it was in the source
it fixed

File: locales/lt-LT/messages.po [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+msgid "Anonymise"
+msgstr "Anonimizuoti"
+
+msgid "Class"
+msgstr "Klasė"
+
+msgid "CSV results export for delivery \"%s"
+msgstr "CSV rezultatų eksportavimas pateikčiai \"%s"
+
+msgid "CSV results export for delivery class \"%s"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Missing closing `"`

File: locales/lt-LT/messages.po [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+msgid "All collected variables"
+msgstr "Visi surinkti kintamieji"
+
+msgid "Anonymise"
+msgstr "Anonimizuoti"
+
+msgid "Class"
+msgstr "Klasė"
+
+msgid "CSV results export for delivery \"%s"
```
[the remark this answers is not in the document]
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: i fixed it

File: locales/lt-LT/messages.po (hunk 2 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

**Pull request merged**